Dalamud/Game/Gui/TextGimmickHint.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+using Dalamud.Plugin.Services;
+
+using FFXIVClientStructs.FFXIV.Client.UI;
+
+using static Dalamud.Plugin.Services.ITextGimmickHint;
+
+namespace Dalamud.Game.Gui
```

Comment:
I think file scoped namespaces are preferred? Did stylecop not complain about this?